osu.Game/Overlays/KeyBinding/KeyBindingsSubsection.cs

```diff
@@ -41,7 +41,7 @@ private void load(KeyBindingStore store)
                 int intKey = (int)defaultGroup.Key;
 
                 // one row per valid action.
-                Add(new KeyBindingRow(defaultGroup.Key, bindings.Where(b => ((int)b.Action).Equals(intKey)))
+                Add(new KeyBindingRow(defaultGroup.Key, bindings.Where(b => b.IntAction == intKey))
```

Comment:
I'm not sure I follow. You've added a new column and prepared migrations, but are still using the old column in all usages?

Also why store both? `KeyBinding`'s `Action` is already an object, so you could just store either an int or string there and match accordingly?

Reply:
I tried nearly everything in that regard and I couldn't get it to work with populating the Action either by an integer value or string dynamically. This is partially because I can't really control in what order EF sets initial values. If you have an Idea, please outline it to me.

Reply:
Also the implementation and usage of the migrated value will follow in a seperate commit but I might try another approach again.

Reply:
Unless